### PR TITLE
Add `script` option

### PR DESCRIPTION
**Pull request by kevva**
@sindresorhus

### CODE REVIEW

**Comment by IonicaBizau** [comment]
:+1:

File: stream.js [hunk 1]
```diff
@@ -117,6 +117,10 @@ page.open(opts.url, function (status) {
 			page.clipRect = clipRect;
 		}
 
+		if (opts.script) {
+			page.evaluateJavaScript('function () { ' + opts.script + '}');
```
Comment (SamVerschueren) [inline stream.js]: Shouldn't this be an IIFE?

```js
page.evaluateJavaScript('(function () { ' + opts.script + '})()');
```

Or does phantom automatically does that?

File: readme.md [hunk 2]
```diff
@@ -77,6 +77,12 @@ Type: `string`
 
 Apply custom CSS to the webpage. Specify some CSS or the path to a CSS file.
 
+##### script
+
+Type: `string`
+
+Apply custom JS to the webpage. Specify some JS or the path to a JS file.
```
Comment (SamVerschueren) [inline readme.md]: I would write `JS` as `JavaScript`.

File: stream.js (hunk 1 above)
Comment (kevva) [inline stream.js]: Yes.

**Comment by IonicaBizau** [comment]
@sindresorhus @SamVerschueren @kevva Thank you, guys! You're amazing. ✨

**Pull request merged**